profile_helper.fish

```diff
@@ -104,6 +104,9 @@ for script_path in $BASE16_SHELL_PATH/scripts/*.sh
   alias $function_name="set_theme \"$theme_name\""
 end
 
+# unset loop variables to not leak to user's shell
+set -e script_path function_name theme_name
```

Comment:
the fish script doesn't define `script_name`
Additionally, I tested and it doesn't seen to expose `script_path` (maybe for-loop variables are automatically destroyed?) but I left it in anyway as it is harmless